editoast/src/views/authz.rs

```diff
@@ -145,6 +152,126 @@ pub(in crate::views) async fn user_groups(
     Ok(Json(result))
 }
 
+#[derive(Deserialize, Clone, ToSchema)]
+pub struct UsersInfoRequest {
```

Comment:
```suggestion
pub(in crate::views) struct UsersInfoRequest {
```

this way we'll be able to just `sed` this when `views` will be a crate